scripts/templates/soccer.wbt.template

```diff
@@ -506,6 +506,21 @@ DEF B1 Robot {
       }
     }
     Receiver {
+      channel 1
+      allowedChannels [1]
+      name "supervisor receiver"
+    }
+    Receiver {
+      channel 2
+      allowedChannels [2]
+      name "team receiver"
+    }
+    Emitter {
+      range 10
```

Comment:
@Adman so if I got it correctly, these are only necessary for the Emitter, right?

Reply:
Yeah, 10 meters for us means basically whole field, so there is no restriction regarding that.